ilrWebServices/ilr-website
# Sdc coures

UPDATE 8/2/2014:
Changes from last meeting:
Never show more than X days in the past from todays - Set to 90

Are keywords being importing? - Working.

For faculty description. if starts with comma. Put name before it. - Fixed.

Show message when that class is full on listing and course page
disable/don't show radio if class is full - Fixed

Change no results text for day - Fixed

Course page:
Remove faculty description
Create link to class node - "Learn more - Fixed

If only one session then show faculty description below form - Fixed

On block nav make sure day links respect current filters - Fixed(but can't use ajax with custom javascript)

---

Contains all SDC 
*content types
*importers
*context
*views
*taxonomy

To test:
Run importer at: http://www.ilr-website.dev/import/sdc_course_importer
- This will automatically run other imports
  -- **\* Importer has timeout issues but I am assuming this is not how the feeds will actually importing feed

View Calendar at  http://www.ilr-website.dev/sdc-catalog-month
- Should see Month/day selection in sidebar
- Click a class should take you to course page with class pre-selected. 
  ---- **\* Course page doesn't look good but this will work out later, right?
- Register button should redirect to current registration system with class sent over.

PS
Sorry for branch name misspelling
